delivery_carrier_label_paazl/models/stock_picking.py

```diff
@@ -67,15 +67,26 @@ def _paazl_generate_labels(self):
         """
         self.ensure_one()
         api = self._paazl_api()
-        # we need to create labels in order to get a tracking number
-        status = api.service.generateLabels(**self._paazl_generate_labels_data())
+        is_return = self.location_id.usage == "customer"
+        return_rma = all([is_return and self.picking_type_code == "incoming"])
+        if return_rma:
+            payload = self._paazl_generate_labels_data()
+            # Removing since generatePdfReturnLabels call doesn't accept labelFormat
+            if "labelFormat" in payload:
+                payload.pop("labelFormat")
```

Comment:
Do we need to pass the shippingOption in the payload here? Presumably this is not the same as the original order's option but an option that triggers a pickup at the customer location.

Reply:
Without shippingOption pass, I wasnt able to make a call to that endpoint, error text was similar to : "Please provide at least one shipping option"